tools/settlement/bitflyer/upload.go

```diff
@@ -248,13 +249,23 @@ func setupSettlementTransactions(
 }
 
 func createBitflyerRequest(
+	ctx context.Context,
 	dryRun *bitflyer.DryRunOption,
 	token string,
 	settlementRequests []settlement.AggregateTransaction,
 ) (*bitflyer.WithdrawToDepositIDBulkPayload, error) {
 	set := []custodian.Transaction{}
 	sourceFrom := ""
+	maxAmountAny := ctx.Value(appctx.PayoutTxnMaxAmountCTXKey)
+	maxAmount, ok := maxAmountAny.(decimal.Decimal)
```

Comment:
Nit: We could perhaps shrink it to:

```suggestion
	maxAmount, ok := ctx.Value(appctx.PayoutTxnMaxAmountCTXKey).(decimal.Decimal)
```